build-system/compile/post-closure-babel.js

```diff
@@ -110,8 +110,8 @@ exports.postClosureBabel = function(directory, isEsmBuild) {
     const {compressed, terserMap} = terserMinify(code);
     file.contents = Buffer.from(compressed, 'utf-8');
 
-    // TODO: Resorcery should support a chain, instead of multiple invocations.
-    remapped = resorcery(
+    // TODO: Remapping should support a chain, instead of multiple invocations.
+    remapped = remapping(
       terserMap,
       returnMapFirst(remapped),
       !argv.full_sourcemaps
```

Comment:
Nit: We can actually fix this comment now, and do only a single remapping: https://github.com/ampproject/remapping#multiple-transformations-of-a-file

So, delete the first remapping call. Here:

```suggestion
    const remapped = remapping(
      [terserMap, babelMap, map],
      loadSourceMap,
      !argv.full_sourcemaps
```

Reply:
Tried to do this change and got an error. Looks like a good followup for a separate PR.

```
TypeError: Cannot read property 'mappings' of null
    at decodeSourceMap (/Users/kristoferbaxter/Projects/amphtml/node_modules/@ampproject/remapping/dist/remapping.umd.js:178:28)
    at Array.map (<anonymous>)
    at buildSourceMapTree (/Users/kristoferbaxter/Projects/amphtml/node_modules/@ampproject/remapping/dist/remapping.umd.js:679:35)
    at remapping (/Users/kristoferbaxter/Projects/amphtml/node_modules/@ampproject/remapping/dist/remapping.umd.js:781:21)
    at DestroyableTransform._transform (/Users/kristoferbaxter/Projects/amphtml/build-system/compile/post-closure-babel.js:78:22)
```